source/fem-dem/gls_vans.cc

```diff
@@ -844,6 +847,189 @@ GLSVANSSolver<dim>::quadrature_centered_sphere_method()
   system_rhs_void_fraction.compress(VectorOperation::add);
 }
 
+template <int dim>
+void
+GLSVANSSolver<dim>::satellite_point_method()
+{
+  QGauss<dim>         quadrature_formula(this->number_quadrature_points);
+  const MappingQ<dim> mapping(1,
```

Comment:
Eventually will be a good idea to use the mapping of the solver itself and not recreate mappings all the time...